.github/workflows/validate.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         id: setup-haskell
         with:
           ghc-version: ${{ matrix.ghc }}
-          cabal-version: latest # default, keeping for visibility
+          cabal-version: '3.8.1.0'
```

Comment:
I missed why now need to specify this at all. Any theories?

Reply:
`latest` is still 3.6, and 9.4 can't run with anything older than 3.8 on Windows. We could petition to the action maintainers to bump latest, but they have big backlog, and it's hard to predict when they'll get around to do that.

Reply:
Got it. Makes sense and we don't want to petition GHA masters each time cabal is released.